bedrock/firefox/templates/firefox/all/download.html

```diff
@@ -0,0 +1,82 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from "macros.html" import google_play_button, apple_app_store_button, ms_store_button with context %}
+
+{% set icon_download ='<span class="mzp-c-button-icon-end"><svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" viewBox="0 0 16 16" width="16" height="16" fill="currentColor"><path d="M8 13c.2 0 .4-.1.5-.2l4.4-4.4-1.1-1.1-3.1 3.1V1H7.2v9.4L4.1 7.3l-1 1.1 4.4 4.4c.1.1.3.2.5.2z" /><path d="M13.5 12v2c0 .3-.2.5-.5.5H3c-.3 0-.5-.2-.5-.5v-2H1v2c0 1.1.9 2 2 2h10c1.1 0 2-.9 2-2v-2h-1.5z" /></svg></span>' %}
+
+{# Show download #}
+
+<h2 class="c-step-name{% if not (product and platform and locale) %} t-step-disabled{% endif %}">
+  4. {{ ftl('firefox-all-download') }}
```

Comment:
**question:** this might be one for the L10n team, but might some locales prefer the step numbers here to be part of the localized string?

Reply:
Oh that's a good note — if they were OL it won't be an issue (UA handles that with lang and css defaults), but truth is when they mimic an ordinal sequence they may need to be localised along with the text in languages that don't use `latn` numbers? (as `arab` `beng` `deva`…)

Reply:
✅  Moved the numbers into the strings.